### ISSUE TITLE
timeline in browse app

### ISSUE THREAD

**Issue by AutryIT** [american-art/autry#70]
David, Please be sure to let us know if you decide to create a timeline in the browse app.  We will want to work with the Gilcrease and others to layer in time related periods or events relevant to Native communities.

**Comment by workergnome** [comment]
Definitely worth thinking about this, even if we're not 100% sure that we're going to implement timelines. 

IN particular, the question I'd want to ask is, "How do I know which contextual timeline to apply to a work?"  I'd like the answer *not* to be, 'if it's a Gilcrease work', but perhaps "If the work was created by a person of nationality X, and was created between YEAR and YEAR, then it could be interesting to see that work in the context of Y events".

**Comment by AutryIT** [comment]
Actually, I would prefer not to filter except on the dates of the designated time period - just like you would do with the presidential terms. Whatever items have a date made in that period, should appear,

**Comment by workergnome** [comment]
I'm not sure I understand—are you suggesting that If I'm showing, say, an Edward Hopper, or a letter in the archives, it would be useful to see it in the context of events relevant to Native communities?

I think the hardest thing about timelines is this contextual switching—I think it should either be done in the context of the work, via nationality, subject, or some other clever idea, or in the context of the _user_, which is out of scope for our project, because we don't know who our users will be. I don't know how to be comprehensive—any view of time will privilege a specific viewpoint, and any attempt to capture multiple viewpoints will only highlight viewpoints that are excluded.  This is why I'd like to find an external source for this context—as my historian friends constantly remind me, as soon as you choose what counts as 'important' things, you're taking positions.

**Comment by AutryIT** [comment]
Yes, we agree that the timeline is not a simple search tool but a contextual framework. At the Autry we don’t think that a presidential timeline provides the right context in which to view the material in this particular database as a whole and especially as it relates to the Native content within it. At first thought, I wanted to layer in some time periods to soften the presidential context, but I agree that this is unlikely to lead us anywhere useful. What would you think of using the material in the database itself using an interface such as a slider timeline?

**Comment by ddegler** [comment]
I suggest we look at a specific example (or a few) and see what data is available that we can use to set context. Not to be pedantic, but at this point I would like examples that show what is possible and what partners should strive for in their data - even if for the future. ;-)

I love the idea of focusing an example on Native communities. It is an interesting perspective for a timeline, although (and particularly because) the time period will be interesting to render.

**Comment by AutryIT** [comment]
David, did you decide how you're going to move forward with the timeline?  I will label this issue after we know some details about how it's going to look.

**Comment by workergnome** [comment]
I have not.  Let's leave it open, and we can discuss it once the timeline's built.
